.github/workflows/psalm-github.yml

```
@@ -0,0 +1,23 @@
+name: Psalm show github
+
+on:
+  pull_request:
```

I'd propose also to add `push` or the comparison to the base branch may fail.

```suggestion
on:
  push:
  pull_request:
```